lib/util/git/config.ts

```diff
@@ -24,3 +25,11 @@ export function simpleGitConfig(): Partial<SimpleGitOptions> {
     },
   };
 }
+
+export function gitTimeoutConfig(): Partial<SimpleGitOptions> {
```

Comment:
why don't you merge this to the normal config above? this makes code unnecessary complicated.